.travis.yml

```diff
@@ -7,7 +7,7 @@ before_install:
   - sudo apt-get update
 
 install:
-  - wget https://github.com/gohugoio/hugo/releases/download/v0.54.0/hugo_extended_0.54.0_Linux-64bit.deb && sudo dpkg -i hugo_extended*.deb
+  - wget https://github.com/gohugoio/hugo/releases/download/v0.58.3/hugo_extended_0.58.3_Linux-64bit.deb && sudo dpkg -i hugo_extended*.deb
```

Comment:
If we need a different hugo version, this should be noted in the README.

Reply:
Reverted this, let's see if the build passes.